CHANGELOG.md

```diff
@@ -1,5 +1,32 @@
 # Changelog
 
+## 2020.11.1 (19 November 2020)
+
+### Fixes
+
+1. Interactive window input prompt does not allow any keyboard input.
+   ([#446](https://github.com/Microsoft/vscode-jupyter/issues/446))
+1. Support opening Notebooks using Native Notebook editor even if the Python extension is not installed.
+   ([#1074](https://github.com/Microsoft/vscode-jupyter/issues/1074))   
+1. Show kernel picker in the interactive window.
+   ([#411](https://github.com/Microsoft/vscode-jupyter/issues/411))   
```

Comment:
No news file for #411?

Reply:
There wasn't one so I just hand edited it. They're not necessary for anything other than to make it easy to produce the changelog.